HuskStandalone/HuskStandalone.py

```diff
@@ -126,7 +129,14 @@ def RenderArgument(self):
         # They mention using the `--disable-dummy-raster-product` husk flag.
 
         arguments.append("--make-output-path")
+        
+        version = self.GetPluginInfoEntryWithDefault( "Version", "" )
+        # Version is always in the form (major.minor) which can be converted to float.
+        if version and  (float(version) < 20):
+            return " ".join(arguments)
+        
         arguments.append("--disable-dummy-raster-product")

```

Comment:
Let's split off after the first dot, we only care about the major version.

And by doing so we also don't hit any issues if for whatever reason someone hacks the logic here so that e.g. they can take patch versions as well, `19.5.325` would then still work.

Plus, reversed the logic so that we don't duplicate the `return` logic but just append the argument only when relevant.

```suggestion
        # We assume no version passed will be latest version, 
        # otherwise version would be in the form (major.minor) 
        # where we only consider the major version.
        if not version or float(version.split(".", 1)[0]) >= 20:
            # Supported only on Houdini 20+. 
            arguments.append("--disable-dummy-raster-product")
```